src/client/pps.go

```diff
@@ -46,6 +46,12 @@ const (
 	// PPSHostPathVolume is the name of the volume that uses the
 	// aforementioned hostpath.
 	PPSHostPathVolume = "pachyderm-worker"
+	// PPSWorkerUserContainerName is the name of the container that runs
+	// the user code to process data.
+	PPSWorkerUserContainerName = "user"
+	// PPSWorkerSidecarContainerName is the name of the sidecar container
+	// that runs alongside of each worker container.
+	PPSWorkerSidecarContainerName = "sidecar"
```

Comment:
I agree that having users see a bunch of `pachd` containers would probably be a bit weird. `sidecar` just seems a little too generic for me, it really just tells you what type of container this is but not much about it's purpose. It also might not be the only sidecar we ever have on the worker Would you be opposed to `pfs-sidecar` or something like that?